### DISCUSSION TITLE
MAS is not opening

### DISCUSSION THREAD

**Discussion by suki-luv** [Monika-After-Story/MonikaModDev#9166] [QA - tech support + general] [answered]
![qwq](https://user-images.githubusercontent.com/100366919/155555180-80b53313-a80a-47fc-a33a-8ec69ccc6fc1.PNG)

please help me- i tried all of the things that would maybe work but nothing happened. its always showing error, im really sorry if this bug is already discovered.

**Comment by ThePotatoGuy** [comment, marked as answer]
have you gone through [these troubleshooting steps](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue)
